src/CMakeLists.txt

```diff
@@ -588,7 +588,7 @@ file(MAKE_DIRECTORY "${CMAKE_BINARY_DIR}/Basilisk/architecture")
 file(MAKE_DIRECTORY "${CMAKE_BINARY_DIR}/Basilisk/simulation")
 file(MAKE_DIRECTORY "${CMAKE_BINARY_DIR}/Basilisk/fswAlgorithms")
 
-if(WIN32 AND (PYTHON_VERSION GREATER 3.8))
+if(WIN32 AND (PYTHON_VERSION VERSION_GREATER 3.8))
```

Comment:
Should this `if` statement be broken up and nested?  Right now if the python check fails, it continues assuming we are using a Unix system.  Rather, if the python check fails then we should throw an error and stop on Windows?

Also, why is this python check here?  In `conanfile.py` we already check that python version 3.8 or newer is installed?  Thus, I'd rather remove this python version checking in Cmake?  It is duplicative checking, and harder to maintain?  @patkenneally, do you see any issue just checking `if(WIN32)` here?

Reply:
Considering Python 3.8 is already a requirement anyway, you are right we could just check for `if(WIN32)`.